documentation/source/user_section/tutorials/non-template-registration/contrast-agnostic-registration/coregistering-t1-t2.rst

```diff
@@ -0,0 +1,30 @@
+Coregistering T1w with T2w
+##########################
+
+Now that we've preprocessed our images, we can align the T1w image with the T2w image.
+
+To align the images, we will coregister them together. In other words, we will compute two different transformations: One to bring the T1w image into the T2w space, and one to bring the T2w image into the T1w space. To perform coregistration, we use the ``sct_register_multimodal`` command.
+
+.. code:: sh
+
+   sct_register_multimodal -i t1_crop.nii.gz -d ../t2/t2_crop.nii.gz \
+                           -param step=1,type=im,algo=dl -qc ~/qc_singleSubj
+
+:Input arguments:
+   - ``-i`` : Source image.
+   - ``-d`` : Destination image.
+   - ``-param`` : Here, we specify ``algo=dl``, a deep learning-based registration algorithm that was trained to be agnostic to image contrasts.
+   - ``-qc`` : Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+:Output files/folders:
+   - ``t1_crop_reg.nii.gz`` : The T1w image registered to the T2w space.
+   - ``t2_crop_reg.nii.gz`` : The T2w image registered to the T1w space.
+   - ``warp_t1_crop2t2_crop.nii.gz`` : The warping field to transform the T1w image to the T2w space.
+   - ``warp_t2_crop2t1_crop.nii.gz`` : The warping field to transform the T2w image to the T1w space.
+
+Once the command has finished, at the bottom of your terminal there will be instructions for inspecting the results using :ref:`Quality Control (QC) <qc>` reports. Optionally, If you have :ref:`fsleyes-instructions` installed, a ``fsleyes`` command will printed as well.
```

Comment:
When I ran the commands from the tutorial, I got:
```
Finished! Elapsed time: 19s
WARNING: Cannot generate QC because it requires destination segmentation.

Done! To view results, type:
fsleyes t1_crop.nii.gz t2_crop_reg.nii.gz &


Done! To view results, type:
fsleyes ../t2/t2_crop.nii.gz t1_crop_reg.nii.gz &
```
So, two observations:
- Apparently we need some more arguments to `sct_register_multimodal` to actually generate the QC report? We could add those missing arguments, or just explain that QC won't be available with the given command.
- There are actually two `fsleyes` commands printed. They're coloured green so they both stand out, hard to miss, but maybe mention that there's one for each direction of the transformation (as explained in the tutorial text at the top of the page).

Reply:
> Apparently we need some more arguments to sct_register_multimodal to actually generate the QC report? We could add those missing arguments, or just explain that QC won't be available with the given command.

Oh! Very good catch. `-qc` does indeed implicitly rely on `-dseg` being supplied:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/1cebd5183ce73aaea8af25f638efb68a3f2f2f54/spinalcordtoolbox/scripts/sct_register_multimodal.py#L427-L430

...Even though `-dseg` isn't actually necessary for the registration command itself. :thinking: 

Adding `-dseg` feels a bit odd given that it's not actually used in the registration itself? But, maybe we can make a note of that in the command description, by saying something like:

```
  - `-dseg`: The segmentation for the destination image. This is used when generating the Quality Control (QC) report to crop around the spinal cord for visual clarity.
```

> There are actually two fsleyes commands printed. They're coloured green so they both stand out, hard to miss, but maybe mention that there's one for each direction of the transformation (as explained in the tutorial text at the top of the page).

Ah! Another good catch. This is very much a side effect of me copying and pasting boilerplate text. :sweat_smile: